feature/settings/src/main/java/com/google/jetpackcamera/settings/SettingsViewModel.kt

```diff
@@ -322,6 +325,23 @@ class SettingsViewModel @Inject constructor(
         )
     }
 
+    private fun getVideoQualityUiState(
```

Comment:
The selected video quality may also not be supported on the other camera lens. You need to disable the default lens flip setting in this case like on line 267 for stabilization or line 236 for FPS.

Reply:
I don't believe this is necessary because in CameraXCameraUseCase, tryApplyVideoQualityConstraints() is added. Upon switching lens, if the video quality is unsupported, it should be set to AUTO. The same mechanism exist for dynamic range

Reply:
As discussed offline, we want the Settings screen to be self-consistent since it is the settings that will be used when the app is launched. So even if we are able to make sure things won't crash, setting the quality to UHD on a camera lens that doesn't support it will mean the app will never start up into a state that matches the settings screen.

Think of the `tryApplyVideoQualityConstraints` as a way to handle incompatibilities in quick settings that we don't want to block access to (such as switching cameras), and anything in the Settings screen should follow all the constraints of other settings in the settings screen.